source/installation_guide/upgrading/from-2.x-to-3.0.rst

```diff
@@ -0,0 +1,37 @@
+=====================================
+Upgrading Dovecot from 2.x to 3.0/2.4
+=====================================
+
+.. contents::
+   :depth: 1
+   :local:
+
+
+About version numbers
+=====================
+
+With 3.0 / 2.4 release Pro and CE release have different major version numbers.
+In documentation any reference to 3.0 applies to 2.4 as well.
+
+Recommended upgrade path
+========================
+
+The recommended upgrade path for 3.0/2.4 is to perform a migration to the version.
```

Comment:
Somehow annoying to read 3.0/2.4 rather than 2.4/3.0 which looks prettier with ascending numbers. :)

Reply:
=)